src/components/ActionButton/ToggleActionButton.ts

```diff
@@ -30,25 +31,24 @@ export class ToggleActionButton extends Component {
          * The attribute would be set like this:
          *
          * ```html
-         * <button class='CoveoToggleActionButton' data-activated-icon='&lt;svg width=&quot;1em&quot; height=&quot;1em&quot;&gt;...&lt;/svg&gt;'></button>
+         * <button class='CoveoToggleActionButton' data-activate-icon='&lt;svg width=&quot;1em&quot; height=&quot;1em&quot;&gt;...&lt;/svg&gt;'></button>
          * ```
          */
-        activatedIcon: ComponentOptions.buildStringOption(),
+        activateIcon: ComponentOptions.buildStringOption({ alias: 'deactivatedIcon' }),
 
         /**
-         * Specifies the button tooltip when the button is activated.
+         * Specifies the button tooltip text displayed to activate the button.
          *
          * Default is the empty string.
          *
          * ```html
-         * <button class='CoveoToggleActionButton' data-activated-tooltip='My activated button tooltip'></button>
+         * <button class='CoveoToggleActionButton' data-activate-tooltip='Activate the feature'></button>
          * ```
          */
-        activatedTooltip: ComponentOptions.buildStringOption(),
+        activateTooltip: ComponentOptions.buildStringOption({ alias: 'deactivatedTooltip' }),
```

Comment:
Do we really want to bother with the alias?
I'd be fine with breaking, JSUI-extension is still sub 0 so, sem-ver wise, everyone that uses it should know that it's at their own risks.

Reply:
:D I almost sent the PR without the backward compatibility support. But, I wasn't happy about consciously introducing a breaking change that could easily be handled.